Documents/VS_Code_Folder/DSA_/DSA-Website/src/components/TreeVisualizer.jsx

```
@@ -0,0 +1,2109 @@
+      currentNode.highlight = true;
+    }
+    
+    results.push({ 
+      tree: highlightedTree, 
+      step: currentStep,
+      line: 12,
+      traversalResult: [...traversalResult]
+    });
+    
+    // Add left child to queue
+    if (node.children && node.children[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)